code/src/nuvla/job/actions/jobs_cleanup.py

```diff
@@ -8,24 +8,22 @@
 @action('cleanup_jobs')
 class JobsCleanupJob(object):
 
-    def __init__(self, executor, job):
+    def __init__(self, _, job):
         self.job = job
-        self.es = executor.es
+        self.nuvla = self.job.api
 
     def cleanup_jobs(self):
         logging.info('Cleanup of completed jobs started.')
 
-        number_of_days_back = 7
-        query_string = 'state:(SUCCESS OR FAILED) AND created:<now-{}d'.format(number_of_days_back)
-        query_old_completed_jobs = {'query': {'query_string': {'query': query_string}}}
-        result = self.es.delete_by_query(index='nuvla-job', body=query_old_completed_jobs)
-
-        if result['timed_out'] or result['failures']:
-            error_msg = 'Cleanup of completed jobs have some failures: {}.'.format(result)
-            logging.warning(error_msg)
-            self.job.set_status_message(error_msg)
+        days_back = 7
+        filter = f"(state='SUCCESS' or state='FAILED') and created<'now-{days_back}d'"
+        ret = self.nuvla.delete_bulk(index='job', filter=filter)
```

Comment:
Delete_bulk parameter resource-type is a positional argument, not a kwargs named index.

Reply:
Fixed in 1350a61